cfgov/teachers_digital_platform/jinja2/teachers_digital_platform/activity_index_page.html

```diff
@@ -59,7 +59,7 @@ <h2>Search for activities</h2>
                         </div>
                     </div>
                     <div class="o-form__input-w-btn_btn-container">
-                        <button class="a-btn">Search</button>
+                        <button class="a-btn" aria-label="Search for activity">Search</button>
```

Comment:
@sonnakim I added this more descriptive label for the search button on https://www.consumerfinance.gov//consumer-tools/educator-tools/youth-financial-education/teach/activities/ Let me know if this should be different.

Reply:
Looks good! Maybe we could just add an article — "Search for **an** activity" @anselmbradford